otter/test/convergence/test_service.py

```diff
@@ -1417,3 +1427,28 @@ def test_multiple_lb_pending(self):
                        desired_lbs=desired_lbs),
                 lb_nodes),
             True)
+
+
+class ConvergenceExecutorTests(SynchronousTestCase):
+    def setUp(self):
+        self.fake_attrs = {
+            'gather': 'g',
+            'plan': 'p',
+            'get_desired_group_state': 'gdgs',
+            'update_cache': 'uc',
+        }
+        self.lse = launch_server_executor
+
+    def test_launch_server_executor(self):
+        attrs = {
```

Comment:
Could you add docstrings to these test cases? We try to have docstrings for all tests